src/proto/index.js.flow

```diff
@@ -357,7 +357,22 @@ declare export class Balance {
     accountId: string;
     current: Money;
     available: Money;
-    static create({accountId: string, current: Money, available: Money}): Balance;
+    updatedAtMs: number;
+    typedBalances: Array<TypedBalance>;
+    static create({
+        accountId: string,
+        current: Money,
+        available: Money,
+        updatedAtMs: number,
+        typedBalances: Array<TypedBalance>,
+    }): Balance;
+    toJSON(): Object;
+}
+declare export class TypedBalance {
```

Comment:
is there a need for it to be an exported class (would the client ever need to create a TypedBalance)? If not we can use `type TypedBalance = {...}` and get rid of create() and toJSON()

Reply:
done